### PR TITLE
OD-172 [Feature] Updated form builder to display date formats using moment locales

### PR DESCRIPTION
**Pull request by AndrRyaz**
@sofiiakvasnevska 

## Issue
OD-172 https://weboo.atlassian.net/browse/OD-172

## Description

- Added date formats to form builder date components

## Screenshots/screencasts
http://somup.com/crf12ZbK34
## Backward compatibility
This change is fully backward compatible.
## Reviewers
@upplabs-alex-levchenko @ivandevupp @YaroslavOvdii

### CODE REVIEW

File: css/builder.css [hunk 1]
```diff
@@ -618,6 +618,10 @@ nav>.btn {
   padding-left: 57px;
 }
 
+.form-control[readonly] {
+  background-color: inherit !important;
```
Comment (squallstar) [inline css/builder.css:622]: please confirm why `!important` is required

File: js/libs/core.js [hunk 2]
```diff
@@ -189,6 +190,14 @@ Fliplet.FormBuilder = (function() {
         return this.showLabel || this.showLabel === undefined;
       };
 
+      component.computed.getLocalFormattedValue = function() {
+        if (!this.value) {
```
Comment (squallstar) [inline js/libs/core.js:194]: why is this block required? the next return is the same as this one

File: css/builder.css (hunk 1 above)
Comment (AndrRyaz) [reply css/builder.css:622]: @squallstar this is because when we adding the disabled property the background color is changing, then i need to change the background to `inherit` color

File: js/libs/core.js (hunk 2 above)
Comment (AndrRyaz) [reply js/libs/core.js]: @squallstar your right thanks, fixed

**Review by squallstar — APPROVED** [review]
@sofiiakvasnevska this is ready to be deployed on **staging** for QA.

- **Risk**: High
- **Affected**: App apps with forms and date fields
- **Verify**: Verify date formats are properly displayed and saved

File: js/components/time.js [hunk 3]
```diff
@@ -59,7 +74,9 @@ Fliplet.FormBuilder.field('time', {
     }
   },
   mounted: function() {
-    var $vm = this;
+    if (Fliplet.Env.is('web') && (this.browserSupport('IE11') || this.browserSupport('Safari'))) {
```
Comment (squallstar) [inline js/components/time.js:77]: why is this here again?

File: js/libs/core.js [hunk 4]
```diff
@@ -170,6 +171,17 @@ Fliplet.FormBuilder = (function() {
         }
       };
 
+      component.methods.browserSupport = function(browserType) {
```
Comment (squallstar) [inline js/libs/core.js:174]: why is this here again?

**Comment by AndrRyaz** [comment]
@squallstar because OF-443 and OD-441 are fixes for this feature and the target branches of this PR is the current

**Comment by squallstar** [comment]
@AndrRyaz that doesn't make any sense, the changes I reviewed have already been merged to master in https://github.com/Fliplet/fliplet-widget-form-builder/pull/377 so they're not expected to be here.

**Comment by AndrRyaz** [comment]
@squallstar i reverted last changes and this PR is clear now as been

**Review by squallstar — APPROVED** [review]
@sofiiakvasnevska this is ready to be deployed on **staging** for QA.

- **Risk**: High
- **Affected**: Studio users with date fields
- **Verify**: Verify the issue is resolved
